app/src/main/java/com/github/steroidteam/todolist/view/NoteSelectionFragment.java

```diff
@@ -119,7 +139,7 @@ public void onNegativeClick() {
         newFragment.show(getParentFragmentManager(), "add_dialog");
     }
 
-    private void removeNote(final UUID id, final int position) {
+    private void removeNote(final Note note, final int position) {
```

Comment:
Maybe a dumb question but why change UUID to Note here ?

Reply:
because of this line `viewModel.removeNote(note.getId(), note.getHeaderID());`
I needed to recover the UUID of the header